Assets/Scripts/Lua/CLRBindings/LuaArenaStatus.cs

```diff
@@ -65,14 +82,6 @@ public class LuaArenaStatus {
     }
 
     public void SetSprite(string filename) {
```

Comment:
I'm looking through old PRs, and I noticed SetSprite() here only affects the inner sprite of the arena, and there's no equivalent for changing the outer sprite.

I'm still unsure why someone would even want to change the arena sprite like that given how it's handled by the engine (constant scaling and stuff, it's only really best suited for a 1x1px sprite, not anything else, as shown in ArenaManager L.342...)

It's much easier for the modder to hide the arena at all times and construct their own instead if they want something fancy.

Reply:
Is it good if we omit SetSprite and go with what we have? What is currently implemented:
1. Arena is now accessible outside waves
2. Arena doesn't reset at the end of a wave, except for size, it keeps its color and visibility
3. Arena's inner and outer parts' color and transparency can be changed with innerColor(32) and outerColor(32)

Reply:
Yep, that's exactly what I wanted. I'll look through the code once more and merge if that's good enough!